src/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ set(HEADERS
 
 if(APPLE)
     list(APPEND SOURCES impl/apple/external_commit_helper.cpp impl/apple/keychain_helper.cpp)
-elseif(REALM_PLATFORM STREQUAL "Android")
+elseif(LINUX OR REALM_PLATFORM STREQUAL "Android")
     list(APPEND SOURCES impl/android/external_commit_helper.cpp)
```

Comment:
It's not Android-specific - we can rework it to say `epoll/external_commit_helper.cpp` or `linux/external_commit_helper.cpp` to just make it more general.